webapp/views.py

```diff
@@ -468,12 +468,26 @@ def engage_page(language, page):
         if not metadata:
             flask.abort(404)
         else:
+            related_pages_metadata = []
+            if "related_urls" in metadata:
+                if metadata["related_urls"].strip() != "":
+                    related_urls = metadata["related_urls"].split(",")
+                    # Only show maximum of 3 related pages
+                    for i in range(3):
+                        url = related_urls[i]
+                        page_metadata = engage_pages.get_engage_page(
+                            url.strip()
+                        )
+                        if page_metadata is not None:
```

Comment:
This is what I would do
```suggestion
                    for url in related_urls[:3]:
                        page_metadata = engage_pages.get_engage_page(
                            url.strip()
                        )
                        if page_metadata is not None:
```